eng/pipelines/dotnet-framework-aspnet.yml

```diff
@@ -0,0 +1,14 @@
+trigger: none
+pr:
+- master
+
+variables:
+- template: variables/common.yml
+- name: manifest
+  value: manifest.aspnet.json
+stages:
+- template: ../common/templates/stages/build-test-publish-repo.yml
+  parameters:
+    ${{ if eq(variables['System.TeamProject'], 'public') }}:
```

Comment:
Not yet, will do so today.

Reply:
Public pipeline looks good.
https://dev.azure.com/dnceng/public/_build/results?buildId=280225

Reply:
Something looks awry with the test results publishing (`Publish test results`).  There are no tests showing in the [`tests` tab](https://dev.azure.com/dnceng/public/_build/results?buildId=280225&view=ms.vss-test-web.build-test-results-tab) of the pipeline
```
[command]E:\A\_work\_tasks\PublishTestResults_0b0f01ed-7dde-43ff-9cbb-e48954daf9b1\2.154.0\modules\TestResultsPublisher.exe @E:\A\_work\_temp\4b9ea580-af55-11e9-8d59-0de6db130c44.txt
No Result Found to Publish 'E:\A\_work\2\s\tests\Microsoft.DotNet.Framework.Docker.Tests\TestResults\dlab14_DDVSOWINAGE123_2019-07-25_20_27_28.trx'.
No build level attachments to publish.
##[section]Finishing: Publish Test Results
```

Reply:
Looks like it's not finding any tests to execute.  From the Test Images task:
```
No test is available in E:\A\_work\2\s\tests\Microsoft.DotNet.Framework.Docker.Tests\bin\Release\netcoreapp2.1\Microsoft.DotNet.Framework.Docker.Tests.dll. Make sure that test discoverer & executors are registered and platform & framework version settings are appropriate and try again.
```

Reply:
I found the issue, I just updated this PR and the AspNet pr to fix the same thing. In this PR I also added back the timeouts and the property 'buildMatrixType'. Not sure if they are needed or not but wanted to keep these files consistent.

Reply:
`buildMatrixType` is needed in order to run tests as part of the build stage as is done in public builds.  The timeout is not needed since all these build jobs finish well within the default timeout of 60 minutes.